### DISCUSSION TITLE
Creating Custom Packaging Workflows on GitHub Actions

### DISCUSSION THREAD

**Discussion by Dorothy-Reyes** [pybamm-team/PyBaMM#3351] [Q&A]
Hi all,
I'm trying to set up a custom packaging workflow using GitHub Actions, but I'm running into issues with the configuration. Can anyone help me create an efficient workflow for building and publishing custom packages on GitHub?

**Comment by brosaplanella** [comment]
See https://github.com/pybamm-team/PyBaMM/discussions/3353#discussioncomment-7032538
